### ISSUE TITLE
Non backwards-compatible changes to the White API

### ISSUE THREAD

**Issue by yazinsai** [Cyber-Duck/white-opencart#1]
Hi, thank you for taking the time to make this plugin!

We've recently integrated with a new banking provider and had to make some non-backwards compatible changes with our API. 

We'll be happy to help you update the plugin to address these changes. 

Most notably:
1. When creating a token, we now pass the open key in the Authentication Header and not as a parameter.
2. The `card` parameter is no longer nested in the token request
3. Charge amounts now use cents (or the smallest currency unit) instead of decimals. (e.g. `amount=1000` is interpreted as `$10.00`)

You can find the latest API documentation at the following link: https://whitepayments.com/docs/api

Here's an example of how to create a charge in PHP, using the new API:

``` php
<?php
# Step 1: Create a customer from the card details
require_once("vendor/autoload.php");
White::setApiKey("test_sec_k_886a1e89b2c0abce479bf"); // Secret key

$customer = White_Charge::create(array(
  'description' => "Amember customer",
  'card' => array(
    'number' => "4242424242424242",
    'exp_month' => 12,
    'exp_year' => 2015,
    'cvc' => '123'
    )
  ));
```

**Comment by threesquared** [comment]
Thanks for letting us know. I will update the plugin soon.

**Comment by threesquared** [comment]
Fixed with 910baceefa75412e5a181c13cdaee87b22e6011e

**Comment by yazinsai** [comment]
Thank you @Ben-Speakman -- this looks great! I just made a minor note on the commit https://github.com/Cyber-Duck/white-opencart/commit/910baceefa75412e5a181c13cdaee87b22e6011e#commitcomment-9708970
